debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/Snapshotter.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql.spi;
+
+import io.debezium.connector.postgresql.PostgresConnectorConfig;
+import io.debezium.relational.TableId;
+
+import java.util.Optional;
+
+public interface Snapshotter {
+    void init(PostgresConnectorConfig config, OffsetState sourceInfo,
```

Comment:
What is the exact info needed from the given config?

Reply:
Currently, that is used for the built-in snapshotters so they can still respect the snapshotSelectOverrides. It still seems like some config values might be useful to be referenced by custom snapshotters as well though, such as doing things with the table whitelist or some other config values.

Reply:
ok.